### PR TITLE
Add Trusted Launch Linux VM templates to Azure quickstart template

### PR DESCRIPTION
**Pull request by prash2611**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Adding quick start template for Trusted Launch Linux VM.

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

File: 101-vm-trustedlaunch-linux/azuredeploy.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     },
     "location": {
       "type": "string",
-      "defaultValue": "[resourceGroup().location]",
+      "defaultValue": "southcentralus",
```
Comment (bmoore-msft) [inline 101-vm-trustedlaunch-linux/azuredeploy.json:79]: you cannot default to a fixed location... if the sample does not work in all locations state that in the readme file and use allowedValues.   If you need a fixed location for the test you can use the param file.

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by bmoore-msft** [comment]
also check the pipeline results for other changes that are needed.

File: 101-vm-trustedlaunch-linux/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+    "networkSecurityGroupName": {
+      "type": "string",
+      "defaultValue": "nsg",
+      "metadata": {
+        "description": "Name of the network security group"
+      }
+    }
+  },
+  "variables": {
+    "osPublisherMap": {
```
[the remark this answers is not in the document]
Comment (prash2611) [reply 101-vm-trustedlaunch-linux/azuredeploy.json]: Done

File: 101-vm-trustedlaunch-linux/azuredeploy.json (hunk 1 above)
Comment (prash2611) [reply 101-vm-trustedlaunch-linux/azuredeploy.json]: Done

**Comment by bmoore-msft** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

File: 101-vm-trustedlaunch-linux/azuredeploy.json (hunk 1 above)
Comment (bmoore-msft) [reply 101-vm-trustedlaunch-linux/azuredeploy.json]: You need set allowedValues to the supported locations

**Comment by bmoore-msft** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Pull request merged**